Driverless AI/ES/automatic-ml-intro-test-drive-tutorial/automatic-ml-intro-test-drive-tutorial.md

```diff
@@ -1,34 +1,34 @@
 
 # Automatic Machine Learning Introduction with Driverless AI
 
-## Outline
+## Contorno
 
-- [Objective](#objective)
-- [Prerequisites](#prerequisites)
-- [Task 1: Product Tour](#task-1-product-tour)
-- [Task 2: Automatic Machine Learning Concepts](#task-2-automatic-machine-learning-concepts)
+- [Objetivo](#objective)
+- [Requisitos Antes de Comenzar](#prerequisites)
```

Comment:
Please update #prerequisites to #requisitos-antes-de-comenzar

Reply:
Please update #Prerequisites to #Prerequisitos, a mi parecer queda más de acuerdo.

Reply:
I changed it to "prerrequisitos" es la forma de escribir _prerequisites_ en español.